DotNetSessionApi64.Example/Program.cs

```diff
@@ -1,59 +1,82 @@
-﻿using System;
+﻿using MAT.AtlasSessionApi;
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
 
-namespace MAT.AtlasSessionApi.TestApp
+namespace DotNetSessionApi64.Example
 {
     internal static class Program
     {
         private static void Main()
         {
-            // The complete path for the session and associated session file names.
-            const string testSsnFileName = @"  \\mes-pwdata01\MESL\OCS\Data\Test Data\SsnFiles\64BitApiTestFiles\Race1_Germany_CH200_D1_153353.ssn   ";
-            string[] associatedSessionFileNames = { @"  Race1_Germany_CH200_D1_153353.VTS.002.ssv  ", @" Race1_Germany_CH200_D1_153353.Test.004.ssv  " };
 
+            const string menu = @"
+            #####################################################################################################
+            
+            Example code showing how to read-write ATLAS session files
+
+            Option 1 - Create a new session file with bespoke parameters
+            Option 2 - Read all parameters from an existing session file (with optional associated sessions)
+            Option 3 - Read selected parameters from an existing session file (with optional associated sessions)
+            Option 4 - Create a new session file with bespoke parameters and then read it
+            
+            #####################################################################################################";
+
+            // ################ Settings for Options 1 & 4 ################
+
+            // The complete path for the session
+            const string fullPathToNewSsn = @"C:\temp\dummy.ssn";
+            
+            // ################ Settings for Options 2 & 3 ################
+
+            // Full file path to the SSN to read
+            const string fullPathToExistingSsn = @"m:\SessionBrowser\SSN\VTS - Copy\Event Compare 180503163009.ssn";
```

Comment:
this path doesn't really mean anything to the outside world - is it overridable from the command line?